qgrid/grid.py

```diff
@@ -20,10 +20,24 @@ def template_contents(filename):
 SLICK_GRID_CSS = template_contents('slickgrid.css.template')
 SLICK_GRID_JS = template_contents('slickgrid.js.template')
 
+SLICK_GRID_DEFAULT_OPTIONS = {
+    'enableCellNavigation': True,
+    'fullWidthRows': True,
+    'syncColumnCellResize': True,
+    'forceFitColumns': True,
+    'rowHeight': 28,
+    'enableColumnReorder': False,
+    'enableTextSelectionOnCells': True, }
+
+
+def show_grid(data_frame, *args, **kwargs):
+    return SlickGrid(data_frame, *args, **kwargs)
+
 
 class SlickGrid(object):
 
-    def __init__(self, data_frame, remote_js=False):
+    def __init__(self, data_frame, remote_js=False, precision=None,
+                 options={}):
```

Comment:
This shouldn't be a dict: see http://docs.python-guide.org/en/latest/writing/gotchas/#mutable-default-arguments